output/influxdb/config_test.go

```diff
@@ -51,3 +52,13 @@ func TestParseURL(t *testing.T) {
 		})
 	}
 }
+
+func TestGetConsolidatedConfig(t *testing.T) {
```

Comment:
```suggestion
func TestGetConsolidatedConfigHTTPProxy(t *testing.T) {
```
The current name is too generic for the current implementation which is very limited to the HTTP proxy.